topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to recurrent neural networks (RNN)
+zenodo_link: https://zenodo.org/record/4477881
+questions:
+- What is a recurrent neural network (RNN)?
+- What are some applications of RNN?
+objectives:
+- Understand the difference between feedforward neural networks (FNN) and RNN
+- Learn various RNN types and architectures
+- Solve a sentiment analysis problem on IMDB movie review dataset using RNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Artificial neural networks are a machine learning discipline roughly inspired by how neurons in a 
+human brain work. In the past decade, there has been a huge resurgence of neural networks thanks 
+to the vast availability of training data and enormous increases in computing capacity (Successfully 
+training complex neural networks in some domains requires lots of data and compute capacity). There 
+are various types of neural networks (Feedforward, recurrent, etc). In this tutorial, we discuss 
+the recurrent neural networks and explain how they differ from feedforward variants. We also describe 
+various RNN architectures and solve a sentiment analysis problem using RNN in Galaxy.       
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Review of feedforward neural networks (FNN)
+
+In feedforward neural networks (FNN) all the fields of a training example are presented
+to the network at once, after which the the network generates an output. For example, a
+lung X-ray image is passed to a FNN, and the network predicts tumor or no tumor. By contrast,
+in RNN the training example fields are presented to the network one at a time. For example,
```

Comment:
Please explain what are training example fields. Do you mean "features"?

Reply:
I'll re-word this so its more clear.